cmd/fyne_demo/main.go

```diff
@@ -181,9 +188,15 @@ func makeNav(setTutorial func(tutorial tutorials.Tutorial), loadPrevious bool) f
 				return
 			}
 			obj.(*widget.Label).SetText(t.Title)
+			if !supportedTutorial(t) {
```

Comment:
These nodes are re-used on scroll, so the "supported" case must reset the italic back to regular